DAO.sol

```diff
@@ -350,6 +349,34 @@ contract DAO is DAOInterface, Token, TokenCreation {
         if (balanceOf(msg.sender) == 0) throw;
             _
     }
+    modifier creationOver{
+          if (now < closingTime) throw;
+	    _
+    }
+    modifier Fueled{
+          if(!isFueled) throw;
+            _
+    }
+    modifier notDAO {
+        if (msg.sender == address(this)) throw;
+            _
+    }
+    modifier onlyDAO {
+        if (msg.sender != address(this)) throw;
+            _
+    }
+    modifier whiteList(address _newContract) {
```

Comment:
Also I think you should remove this too. Because:
1) It's used in only one place as a modifier.
2) Modifiers that take arguments make the code hard to read, in my opinion